### PR TITLE
o [NEXUS-5193] add IT for ldaps:// connections

### PR DESCRIPTION
**Pull request by nabcos**
https://builds.sonatype.org/job/nexus-oss-its-feature/381/

### CODE REVIEW

File: nexus/nexus-core-plugins/nexus-ldap-plugin-parent/nexus-ldap-plugin-it/src/test/java/org/sonatype/nexus/security/ldap/realms/testharness/nexus5193/TrustingX509TrustManager.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ */
+package org.sonatype.nexus.security.ldap.realms.testharness.nexus5193;
+
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+import com.sun.net.ssl.internal.ssl.X509ExtendedTrustManager;
+
+/**
+* A very naive TrustManager.
```
Comment (nabcos) [inline nexus/nexus-core-plugins/nexus-ldap-plugin-parent/nexus-ldap-plugin-it/src/test/java/org/sonatype/nexus/security/ldap/realms/testharness/nexus5193/TrustingX509TrustManager.java]: > @peterlynch: would something this generic be better suited for litmus?

See sonatype/sisu-litmus#7, but can we upgrade to newest litmus in Nexus? (IIRC there was a problem with junit?)

**Comment by peterlynch** [comment]
+1

File: nexus/nexus-core-plugins/nexus-ldap-plugin-parent/nexus-ldap-plugin-it/src/test/java/org/sonatype/nexus/security/ldap/realms/testharness/nexus5193/TrustingX509TrustManager.java (hunk 1 above)
Comment (peterlynch) [inline nexus/nexus-core-plugins/nexus-ldap-plugin-parent/nexus-ldap-plugin-it/src/test/java/org/sonatype/nexus/security/ldap/realms/testharness/nexus5193/TrustingX509TrustManager.java]: There will be no problem in the next few days

**Comment by cstamas** [comment]
+1

**Comment by adreghiciu** [comment]
+1

**Comment by peterlynch** [comment]
ok I updated to litmus trust manager

**Comment by nabcos** [comment]
https://builds.sonatype.org/job/nexus-oss-its-feature/403/

**Comment by nabcos** [comment]
CI had an unrelated failure in the feature build

**Pull request merged**